### PR TITLE
traducción al español de  token_classification.mdx

### PR DESCRIPTION
**Pull request by gpalomeque**
# What does this PR do?

Fixes # 15947

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/es/_toctree.yml.bak [hunk 1]
```diff
@@ -0,0 +1,44 @@
+- sections:
```
Comment (omarespejel) [inline docs/source/es/_toctree.yml.bak:1]: This file, `_toctree.yml.bak`, can be erased.

File: docs/source/es/tasks/token_classification.mdx [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Clasificación de tokens
+
+<Youtube id="wVHdVlPScxA"/>
+
+La clasificación de tokens asigna una etiqueta a los tokens individuales de una frase. Una de las tareas más comunes de clasificación de tokens es el Reconocimiento de Entidades Nombradas (en inglés, NER). El NER intenta encontrar una etiqueta para cada entidad en una frase, como persona, ubicación, u organización.
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:17]:
```suggestion
En la clasificación de tokens se asigna una etiqueta a los tokens individuales de una frase. Una de las tareas más comunes de clasificación de tokens es la de Reconocimiento de Entidades Nombradas (en inglés, Name Entity Recognition o NER). El NER intenta encontrar una etiqueta para cada entidad en una frase como una persona, ubicación, u organización.
```

File: docs/source/es/tasks/token_classification.mdx [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
+Esta guía te mostrará cómo afinar el modelo (en inglés, finetunning) [DistilBERT](https://huggingface.co/distilbert-base-uncased) con el conjunto de datos [WNUT 17](https://huggingface.co/datasets/wnut_17) para detectar nuevas entidades.
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:19]:
```suggestion
Esta guía te mostrará cómo afinar (en inglés, fine-tuning) el modelo [DistilBERT](https://huggingface.co/distilbert-base-uncased) con el conjunto de datos [WNUT 17](https://huggingface.co/datasets/wnut_17) para detectar nuevas entidades.
```

File: docs/source/es/tasks/token_classification.mdx [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+
+<Tip>
+
+Consulta la [página de la tarea] de clasificación de tokens (https://huggingface.co/tasks/token-classification) para obtener más información sobre otras formas de clasificación de tokens y sus modelos, conjuntos de datos y métricas asociadas.
+
+</Tip>
+
+## Cargar el conjunto de datos WNUT 17
+
+Cargue el conjunto de datos WNUT 17 de la biblioteca 🤗 Datasets :
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:29]:
```suggestion
Carga el conjunto de datos WNUT 17 de la biblioteca 🤗 Datasets :
```

File: docs/source/es/tasks/token_classification.mdx [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+
+Configura el modelo para el entrenamiento con [`compile`](https://keras.io/api/models/model_training_apis/#compile-method):
+
+```py
+>>> import tensorflow as tf
+
+>>> model.compile(optimizer=optimizer)
+```
+
+Llame a [`fit`](https://keras.io/api/models/model_training_apis/#fit-method) para afinar el modelo:
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:256]:
```suggestion
Llama a [`fit`](https://keras.io/api/models/model_training_apis/#fit-method) para afinar el modelo:
```

File: docs/source/es/tasks/token_classification.mdx [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+... )
+```
+
+<Tip>
+
+Si no estas familiarizado con el ajuste fino de un modelo con Keras, da un vistazo al tutorial básico [aquí](training#finetune-with-keras)!
+
+</Tip>
+
+Configura una función de optimización, un programa de tasa de aprendizaje y algunos hiperparámetros de entrenamiento:
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:224]:
```suggestion
Configura una función de optimización, un scheduler de tasa de aprendizaje y algunos hiperparámetros de entrenamiento:
```

File: docs/source/es/tasks/token_classification.mdx [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+...     columns=["attention_mask", "input_ids", "labels"],
+...     shuffle=False,
+...     batch_size=16,
+...     collate_fn=data_collator,
+... )
+```
+
+<Tip>
+
+Si no estas familiarizado con el ajuste fino de un modelo con Keras, da un vistazo al tutorial básico [aquí](training#finetune-with-keras)!
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:220]:
```suggestion
Si no estas familiarizado con el afinamiento de un modelo con Keras, da un vistazo al tutorial básico [aquí](training#finetune-with-keras)!
```

File: docs/source/es/tasks/token_classification.mdx [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+...     eval_dataset=tokenized_wnut["test"],
+...     tokenizer=tokenizer,
+...     data_collator=data_collator,
+... )
+
+>>> trainer.train()
+```
+</pt>
+<tf>
+Para ajustar un modelo en TensorFlow, empieza por convertir tus conjuntos de datos al formato `tf.data.Dataset` con [`to_tf_dataset`](https://huggingface.co/docs/datasets/package_reference/main_classes.html#datasets.Dataset.to_tf_dataset). Especifica las entradas y las etiquetas en `columns`, si deseas mezclar el orden del conjunto de datos, el tamaño del lote y el lote de ejemplos:
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:200]:
```suggestion
Para afinar un modelo en TensorFlow empieza por convertir tus conjuntos de datos al formato `tf.data.Dataset` con [`to_tf_dataset`](https://huggingface.co/docs/datasets/package_reference/main_classes.html#datasets.Dataset.to_tf_dataset). Especifica las entradas y las etiquetas en `columns`, si deseas mezclar el orden del conjunto de datos, el tamaño del lote y el lote de ejemplos:
```

File: docs/source/es/tasks/token_classification.mdx [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+>>> model = AutoModelForTokenClassification.from_pretrained("distilbert-base-uncased", num_labels=14)
+```
+
+<Tip>
+
+Si no estas familiarizado con el ajuste fino de un modelo con el [`Trainer`], da un vistazo al tutorial básico [aquí](../training#finetune-with-trainer)!
+
+</Tip>
+
+En este punto, solo quedan tres pasos:
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:170]:
```suggestion
En este punto solo quedan tres pasos:
```

File: docs/source/es/tasks/token_classification.mdx [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+
+1. Definir los hiperparámetros del entrenamiento en [`TrainingArguments`].
+2. Pasar los parámetros de entrenamiento a [`Trainer`] junto con el modelo, el conjunto de datos, el tokenizador y el lote de ejemplos.
+3. Llamar [`~Trainer.train`] para ajustar el modelo.
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:174]:
```suggestion
3. Llamar [`~Trainer.train`] para afinar el modelo.
```

File: docs/source/es/tasks/token_classification.mdx [hunk 11]
```diff
@@ -0,0 +1,270 @@
...
+
+<Tip>
+
+Si no estas familiarizado con el ajuste fino de un modelo con el [`Trainer`], da un vistazo al tutorial básico [aquí](../training#finetune-with-trainer)!
+
+</Tip>
+
+En este punto, solo quedan tres pasos:
+
+1. Definir los hiperparámetros del entrenamiento en [`TrainingArguments`].
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:172]:
```suggestion
1. Definir los hiperparámetros de entrenamiento en [`TrainingArguments`].
```

File: docs/source/es/tasks/token_classification.mdx [hunk 12]
```diff
@@ -0,0 +1,270 @@
...
+
+```py
+>>> from transformers import AutoModelForTokenClassification, TrainingArguments, Trainer
+
+>>> model = AutoModelForTokenClassification.from_pretrained("distilbert-base-uncased", num_labels=14)
+```
+
+<Tip>
+
+Si no estas familiarizado con el ajuste fino de un modelo con el [`Trainer`], da un vistazo al tutorial básico [aquí](../training#finetune-with-trainer)!
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:166]:
```suggestion
Si no estas familiarizado con el afinamiento de un modelo con el [`Trainer`], da un vistazo al tutorial básico [aquí](../training#finetune-with-trainer)!
```

File: docs/source/es/tasks/token_classification.mdx [hunk 13]
```diff
@@ -0,0 +1,270 @@
...
+...             else:
+...                 label_ids.append(-100)
+...             previous_word_idx = word_idx
+...         labels.append(label_ids)
+
+...     tokenized_inputs["labels"] = labels
+...     return tokenized_inputs
+```
+
+Utiliza la función 🤗 Datasets [`map`](https://huggingface.co/docs/datasets/package_reference/main_classes.html#datasets.Dataset.map) para tokenizar y alinear las etiquetas en todo el conjunto de datos. Puedes acelerar la función `map` estableciendo `batched=True` para procesar varios elementos del conjunto de datos a la vez:
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:127]:
```suggestion
Utiliza la función [`map`](https://huggingface.co/docs/datasets/package_reference/main_classes.html#datasets.Dataset.map) de 🤗 Datasets  para tokenizar y alinear las etiquetas en todo el conjunto de datos. Puedes acelerar la función `map` estableciendo `batched=True` para procesar varios elementos del conjunto de datos a la vez:
```

File: docs/source/es/tasks/token_classification.mdx [hunk 14]
```diff
@@ -0,0 +1,270 @@
...
+['[CLS]', '@', 'paul', '##walk', 'it', "'", 's', 'the', 'view', 'from', 'where', 'i', "'", 'm', 'living', 'for', 'two', 'weeks', '.', 'empire', 'state', 'building', '=', 'es', '##b', '.', 'pretty', 'bad', 'storm', 'here', 'last', 'evening', '.', '[SEP]']
+```
+
+La incorporación de los tokens especiales `[CLS]` y `[SEP]` y la tokenización de subpalabras crea un desajuste entre la entrada y las etiquetas. Una sola palabra que corresponde a una sola etiqueta puede estar dividida en dos subpalabras. Tendrás que volver a alinear los tokens y las etiquetas:
+
+1. Asignar todos los tokens a la palabra correspondiente con el método [`word_ids`](https://huggingface.co/docs/tokenizers/python/latest/api/reference.html#tokenizers.Encoding.word_ids).
+2. Asignar la etiqueta `-100` a los tokens especiales `[CLS]` y `[SEP]` para que la función de perdida de PyTorch los ignore.
+3. Etiquetar sólo el primer token de una palabra dada. Asignar `-100` a otros subtokens de la misma palabra.
+
+Así es como se puede crear una función para realinear los tokens y las etiquetas, y truncar las secuencias para que no sean más largas que la longitud máxima de entrada de DistilBERT::
```
Comment (omarespejel) [inline docs/source/es/tasks/token_classification.mdx:102]:
```suggestion
Así es como se puede crear una función para realinear los tokens y las etiquetas, y truncar las secuencias para que no sean más largas que la longitud máxima de entrada de DistilBERT:
```

**Review by omarespejel — CHANGES_REQUESTED** [review]

**Comment by omarespejel** [comment]
Hola @gpalomeque! Thank you for your translations. I am sorry for the late review on my side.

I left you some comments on the review. Also, please update your local version of the repository with `git pull upstream main` (you might have to fix some conflicts, please let me know if I can help). This I believe would let your PR pass all tests.

Fixes #15947
